repositories.yaml

```diff
@@ -496,3 +496,25 @@ applications:
     channels:
       - v1_name: firefox-klar-ios
         app_id: org.mozilla.ios.Klar
+
+  - app_name: bergamot
+    canonical_app_name: Bergamot Translator
+    app_description: Web extension for on-device machine translation in Firefox
+    url: https://github.com/mozilla-extensions/bergamot-browser-extension
+    notification_emails:
+      - epavlov@mozilla.com
+    branch: main
+    metrics_files:
+      - extension/ts/background-scripts/background.js/telemetry/metrics.yaml
+    ping_files:
+      - extension/ts/background-scripts/background.js/telemetry/pings.yaml
+    dependencies:
+      - glean-core
+    retention_days: 180
+    channels:
+      - v1_name: bergamot
+        app_id: org.mozilla.bergamot
+        channel: firefox
+      - v1_name: bergamot-cross-browser
```

Comment:
@fbertsch now I'm thinking that if those are channels, it would make sense to split this one into `cross-browser-firefox` and `cross-browser-chrome`. Also gleans.js doesn't add browser name in client info at the moment, so it would help us to distinguish them.

Reply:
That makes sense to me, if it's not a problem on the client end. I'm assuming this hasn't shipped to anyone yet then?

Reply:
No, I'm just clicking locally to send some data) I already changed it on the client side.  It will be shipped very soon though